content/en/network_performance_monitoring/installation.md

```diff
@@ -24,14 +24,19 @@ Supported **platforms** include:
 - SUSE 15+
 - Amazon AMI 2016.03+
 - Amazon Linux 2
+- [Windows 2016+][8] (in public beta)
 
-There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
+**For Linux OS:** Data collection is done using eBPF, so Datadog minimally requires platforms that have an underlying Linux kernel versions of 4.4.0+.
+
+**For Windows OS:** Data collection is available in public beta for Windows versions 2016 or later.
+
+There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6. 
 
 Network Performance Monitoring is compatible with **Cilium** installations, provided the following requirements are met:
 1) Cilium version 1.6 and above, and
 2) Kernel version 5.1.16 and above, or 4.19.57 and above for 4.19.x kernels
 
-**Note**: Datadog does not currently support Windows and macOS platforms for Network Performance Monitoring.
+**Note**: Datadog does not currently support macOS platforms for Network Performance Monitoring.
```

Comment:
```suggestion
**Note**: Datadog does not support macOS platforms for Network Performance Monitoring.
```

Reply:
My concern is a customer will see this and think "NPM will never support my platform" where we might actually support this in the future if there is high demand. Is the edit for consistency w/other docs?

Reply:
Yes. The documentation is only suppose to state what is currently true. We don't state or imply future intent.